Gruntfile.js

```diff
@@ -312,7 +312,7 @@ module.exports = function(grunt) {
         tasks: ['js']
       },
       content: {
-        files: ['**/*.html', '/downloads/*.md', '/guides/*.md', '/identity/*.md', '/ui-toolkit/*.md'],
+        files: ['index.html', '**/*.md'],
```

Comment:
They're not being converted, just triggering the copy task. The main use case would be adding new images to a page. On saving the MD file, images will be recopied from `./src/static/img/` (everything in `src` is ignored by Jekyll) to `./static/img`, where they'll be picked up by Jekyll's watcher.

We could consider altering/getting rid of the `src`/"`dist`" model that's in use here and just keep static files that need no building in `./static`.